.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# Default owner
+* @hashicorp/team-ip-compliance
```

Comment:
Github marks this as being invalid. :thinking:

Reply:
@schmichael We have remediated the issue. Kindly review this PR and merge it.